CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# 5.0.0 (not released yet)
+
+Backwards incompatible:
+
+- Changed `Option` and `Result` iterator behavior such that iterating `Some` and `Ok` will
+  instead produce only one result – the wrapped value. Previously the iteration depended on
+  the type of the wrapped value (iteratable or not) and produced results obtained by iterating
+  the wrapped values.
```

Comment:
nitpick: I would maybe add that iterating over `None` and `Err` produces no results.

Reply:
Scroll two paragraphs down ;)